docs/general/fields.md

```diff
@@ -0,0 +1,23 @@
+[Back to Guides](../README.md)
+
+# Fields
+
+If for a reason, you need to override the fields defined on serializer, you should use `fields`
+
+For example, if you have a serializer like this
+
+```ruby
+class UserSerializer < ActiveModel::Serializer
+  attributes :access_token, :first_name, :last_name
+end
+```
+
+and in a specific controller, you can't return the `first_name`, `fields` will help you
+
+```ruby
+class AnonymousController < ApplicationController
+  def create
+    render json: User.create(activation_state: 'anonymous'), fields: [:access_token], status: 201
```

Comment:
```ruby
def create
  if user = User.create(activation_state: 'anonymous')
    # using the :json adapter
    render json: user, fields: [:access_token], adapter: :json, status: :created
    # using the :json_api adapter
    render json: user, fields: { users: [:access_token] }, adapter: :json_api, status: :created
    # using the registered jsonapi renderer
    render jsonapi: user, fields: { users: [:access_token] }, status: :created
```

@groyoh question: when I serialize a collection with JSONAPI I might have

```ruby
      render_options[:fields] = {items: [:id, :title, :description], locations: [:name]}
      render_options[:include] = [:location]
```

but I'm not sure how this would look with the :json adapter  (lazy me did not review code or check for tests)